Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -490,7 +490,7 @@ script:
       - 'false'
     - description: When specified, the results returned in the list are limited to assets with the specified tag category.
       name: tagCategory
-    - description: When specified, the results returned in the list are limited to assets with the specified tag value.
+    - description: When specified, the results returned in the list are limited to assets with the specified tag value. Can be comma-delimited for multiple items.
```

Comment:
I see that you edited the description and added the `isArray` for different commands, is it on purpose?